cobalt/browser/web_module.cc

```diff
@@ -1130,7 +1142,7 @@ void WebModule::Impl::Conceal(render_tree::ResourceProvider* resource_provider,
 
 #if defined(ENABLE_DEBUGGER)
   // The debug overlay may be holding onto a render tree, clear that out.
-  debug_overlay_->ClearInput();
+  if (debug_overlay_) debug_overlay_->ClearInput();
```

Comment:
I think we always use braces: go/cstyle#Formatting_Looping_Branching

Reply:
This matches the exception in the cstyle because it fits on one line.